sklearn/utils/multiclass.py

```diff
@@ -155,6 +155,18 @@ def is_multilabel(y):
                                     _is_integral_float(labels))
 
 
+def is_onehot(y):
```

Comment:
We will not need that.

Reply:
Hi Guillaume, 
agree but please see https://github.com/scikit-learn/scikit-learn/pull/11007#discussion_r184127587

Reply:
removed.